addons/l10n_ro/partner_view.xml

```diff
@@ -1,15 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <openerp>
-	<data>
-		<record model="ir.ui.view" id="res_partner_form_ro">
-			<field name="name">res.partner.form.ro</field>
-			<field name="inherit_id" ref="base.view_partner_form"/>
-			<field name="model">res.partner</field>
-			<field name="arch" type="xml">
-				<field name="property_account_position" position="after">
-					<field name="nrc" placeholder="e.g. J12/1234/2012" class="oe_inline"/>
+    <data>
+        <record model="ir.ui.view" id="res_partner_form_ro">
+            <field name="name">res.partner.form.ro</field>
+            <field name="inherit_id" ref="account.view_partner_property_form"/>
+            <field name="model">res.partner</field>
+            <field name="arch" type="xml">
+                <field name="property_account_position" position="after">
+                    <field name="nrc" placeholder="e.g. J12/1234/2012" class="oe_inline"/>
                 </field>
```

Comment:
The 11 line was not fixed in the merge...and i made the pr long time ago...before the spotted inheritance mistake.
Should i leave like this or rebase with the last merge.

https://github.com/odoo/odoo/blob/436fc5b67c36d5a4e5a4dfefe56e13239d962c92/addons/l10n_ro/partner_view.xml